### DISCUSSION TITLE
Qn Regarding - QKD Key retrieval from Keyserver for Strongswan

### DISCUSSION THREAD

**Discussion by anoop-va83** [strongswan/strongswan#2296] [General]
Hi,

  I have question regarding QKD based IPSec(strongswan)implementation based on RFC8784. Do we support any kind of Key retriveal mechanisms like ETSI 014, in strongswan?

**Comment by tobiasbrunner** [comment]
No, nothing like that is supported out of the box. I suppose a plugin could provide a SHARED_PPK secret from such a source.
